### DISCUSSION TITLE
My stats does not seems accurate

### DISCUSSION THREAD

**Discussion by lecastus** [anuraghazra/github-readme-stats#3510] [Q&A] [answered]
### Describe the bug

I'm having a similar issue. 

As shown in the images bellow, I have 269 commits this year (2023) but the git stats only recognize 26 of those. The same goes for the most used languages used. I have no data, even though I have nearly 2k commits in many languages. 

Could the team help about that, please?

### Screenshots / Live demo link

![Screenshot_1](https://github.com/anuraghazra/github-readme-stats/assets/62342278/c84d24e9-8796-41af-9b4e-42c211b3b010)
![Screenshot_2](https://github.com/anuraghazra/github-readme-stats/assets/62342278/67e71c3e-8211-4ff9-9ab7-1e10934f97bd)

**Comment by qwerty541** [comment, marked as answer]
Hey, @lecastus! Currently i see only 29 contributions on the graph in your profile, it means that most of your contributions are private. Our documentation clearly says that by default stats card shows only public contributions. To show your private contributions you should [deploy own instance](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own) using own GitHub API token.

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/24ec7003-feea-46e5-9e91-14ec15ca60b6)

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/47035116-4b5a-4807-bb53-bf5f48153a02)
